components/Buttons/yellowStartTestButton.tsx

```diff
@@ -0,0 +1,11 @@
+import scss from './buttons.module.scss';
+import {NextComponentType} from 'next';
+import React from "react";
+import TestPage from "../../pages/testpage";
+
+function RouteToTest(): JSX.Element {
+    return <TestPage/>
+}
+const YellowStartTestButton: NextComponentType = () =>
+                <button title={"start-button"}  className={scss.buttonYellow} onClick={RouteToTest}>Start</button>;
```

Comment:
Isn't this really a link to another page? if so it shouldn't be a button - better for it to actually be a link!